android/build.gradle

```diff
@@ -160,7 +160,19 @@ dependencies {
   extractJNI("com.facebook.fbjni:fbjni:+")
 
   if (!sourceBuild) {
-    def rnAAR = fileTree("${defaultDir.toString()}").matching({ it.include "**/**/*.aar" }).singleFile
+    def buildType = "debug"
+    tasks.all({ task ->
+        if (task.name == "buildCMakeRelease") {
+            buildType = "release"
+        }
+    })
+    def rnAarMatcher = "**/react-native/**/*${buildType}.aar"
+    def (major, minor, patch) = reactProperties.getProperty("VERSION_NAME").tokenize('.')
+    def rnMinorVersion = Integer.parseInt(minor)
+    if (rnMinorVersion < 69) {
```

Comment:
I think you can use `REACT_NATIVE_VERSION` variable that defined above

Reply:
I applied your suggestion. looks much better! thanks👍